x-pack/plugins/apm/public/components/app/service_overview/service_overview_errors_table/index.tsx

```diff
@@ -152,7 +153,7 @@ export function ServiceOverviewErrorsTable({ serviceName }: Props) {
     },
     status,
   } = useFetcher(() => {
-    if (!start || !end) {
+    if (!start || !end || !transactionType) {
```

Comment:
For readability, something like:
```ts
const hasRequiredParams = start && end && transactionType;
if (!hasRequiredParams) {
  return;
}
```

Reply:
We are already using the "or not" style so I think it's fine to keep it. At least that's consistent